tests/test_publish.py

```diff
@@ -1,19 +1,20 @@
 import filecmp
+import tempfile
 import os
 import re
 import shutil
 
-import numpy as np
-import pandas as pd
 import pytest
 
 import audbackend
 import audeer
 import audformat.testing
 import audiofile
-
 import audb
 
+import numpy as np
+import pandas as pd
```

Comment:
Why did you moved those imports down?

Usually the order is:
* build-in packages
* 3rd party packages
* audeering packages
* package under test

Reply:
I thought we were using a different ordering, I've revert it to how it was.